package.json

```diff
@@ -65,6 +65,7 @@
     "riot-route": "^3.1.4",
     "s3-files": "^2.0.3",
     "sanitize-filename": "^1.6.3",
+    "stylelint-config-standard-scss": "^5.0.0",
```

Comment:
is this a dev dependency?

Reply:
I am not sure how that happened.  It was a dev dependency before.  I will move it back to dev dependency.